### PR TITLE
minor copy edit

### PR DESCRIPTION
**Pull request by kwijibo**

### CODE REVIEW

File: docs/spec.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ map :: Functor f => Type f ~> (a → b, f a) → f b
 ```
 
 We use syntax similar to Haskell's. You can learn about it from
-[Ramda's wiki](https://github.com/ramda/ramda/wiki/Type-Signatures) or from book
+[Ramda's wiki](https://github.com/ramda/ramda/wiki/Type-Signatures) or from the book
 ["Professor Frisby's Mostly Adequate Guide to Functional Programming"](https://drboolean.gitbooks.io/mostly-adequate-guide/content/ch7.html)
```
Comment (davidchambers) [inline docs/spec.md]: Let's also add a full stop (`.`) at the end of the sentence. :)

Comment (rpominov) [inline docs/spec.md]: +1
I tend to not add a `.` after a link sometimes because it looks ugly to me. But I guess that's just silly :)

Comment (rpominov) [inline docs/spec.md]: Added it.

**Comment by rpominov** [comment]
Thanks for the PR, @kwijibo !

**Pull request merged**